Sources/DeckUI/Views/Presenter.swift

```diff
@@ -10,30 +10,31 @@ import SwiftUI
 public struct Presenter: View {
     public typealias DefaultResolution = (width: Double, height: Double)
     
+    @ObservedObject private var presentationState = PresentationState.shared
```

Comment:
Using a shared instance here makes it impossible to have two `Presenter`s with different state. Is that intentional? I could imagine that someone might have a use case for having multiple presenters.

Reply:
I'm sure my use cases would be a bit contrived. 😊 
I just wanted to make sure that it is intentional that you can't have two of these on-screen simultaneously.

Reply:
Thanks for including the option to have presenter notes, @zhbrass, I love it! I was able to include presenter notes thanks to your PR.

I'd like to come up with a use case where there is a need to have multiple decks though.
I am currently working on a project to construct a curriculum for a class.
Key features of that app are:
- create class events with specific content
- create content with markdown
- event's content will contain a script and slides, where the slides is what is being presented and the script is what is being handed out to students
- the content gets parsed to a script pdf (preferably containing the slides as images) and a DeckUI deck that I can present (this already partially works in my app)
- as a class consists of more that one event, I'd like to be able to start any of my slides individually

All this is just to say: there are use cases for more than one deck in an app. I am not quite sure yet whether this affects me because my app is still pretty much work in progress though...